tableTheory/tableTheory.tex

```diff
@@ -309,17 +349,17 @@ \subsubsection{Bag Count}
 \begin{itemize}
 \item
 We do not include an explicit membership predicate for bags, although note that
-the formula $\syntax{(>\ (count\ u\ t)\ 0)}$ holds exactly when $\syntax{u}$
+the formula $\syntax{(>\ (bag.count\ u\ t)\ 0)}$ holds exactly when $\syntax{u}$
 is a member of $\syntax{t}$ (with any multiplicity).
 \end{itemize}
 
 \subsubsection{Bag Compositions}
 \label{sec:bag-compositions}
 The signature $\sigtable$ includes 
 the binary function symbols 
-$\syntax{union\_max}$, $\syntax{union\_disjoint}$, 
-$\syntax{intersect\_min}$, $\syntax{difference\_subtract}$ and
-$\syntax{difference\_remove}$.
+$\syntax{bag.union\_max}$, $\syntax{bag.union\_disjoint}$, 
+$\syntax{bag.intersect\_min}$, $\syntax{bag.difference\_subtract}$ and
```

Comment:
This is unfortunate, but in cvc5, we have `bag.inter_min`, not `bag.intersect_min`.

Reply:
Not a big deal, we can change this now.